Kickstarter-iOS/Views/Controllers/PledgeViewControllerTests.swift

```diff
@@ -402,11 +426,15 @@ final class PledgeViewControllerTests: TestCase {
   }
 
   func testView_ChangePaymentMethodContext_UnavailableStoredCards() {
-    let userEnvelope = UserEnvelope(me: GraphUserCreditCard.withCards([
-      GraphUserCreditCard.visa,
-      GraphUserCreditCard.masterCard
-    ]))
-    let mockService = MockService(fetchGraphCreditCardsResponse: userEnvelope)
+    let cardTemplate = GraphUserCreditCard(
+      nodes: [
+        GraphUserCreditCard.visa,
+        GraphUserCreditCard.masterCard
+      ]
+    )
+    let withCards = GraphUser.template |> \.storedCards .~ cardTemplate
+    let response = UserEnvelope<GraphUser>(me: withCards)
+    let mockService = MockService(fetchGraphUserResponse: response)
```

Comment:
Can reduce a lot of code duplication and reference the same `response` in every change on this file.

Reply:
Created a private constant for userWithCards which uses the GraphUser.template and sets the storedCards but I can't store the `response` because it references self so that will have to be re-created for each test method.